Style Fixes

Moobien opened the pull request:
Closes #
Closes #

# Changes proposed in this pull request

1: removed pagoda alt citizen from warped style
2: rescanned supplyship/nethership/ship/camp/supplycamp
3: added nethership alongside supply ship

## Testing
- [ ] Yes I tested this before submitting it.
- [ ] I also did a multiplayer test.

Review please

Moobien committed:
Tag update

Removed extraneous sugar tags

Moobien committed:
Merge branch 'version/main' into Moobien

Moobien committed:
Missing Building Additions

Added Medieval Oak, Medieval Dark Oak and Medieval Birch nether mines courtesy of FourNation_UK

Moobien committed:
Merge branch 'version/main' into Moobien

Moobien committed:
Moob Additions/Edits

Added Missing Medieval Alchemist Towers courtesy of FourNation_UK

Moobien committed:
Merge branch 'version/main' into Moobien

Moobien committed:
Style Fixes

1: removed pagoda alt citizen from warped style
2: rescanned supplyship/nethership/ship/camp/supplycamp
3: added nethership alongside supply ship

MotionlessTrain commented:
I added the warped nether ship in #10648 
Not sure why that is not reflected here (it would be an update rather than an addition)

Moobien committed:
Style Fixes

rescan of original shaft/nodes as the ones in the mod don't have the tag anchors marking them as invisible.

Raycoms commented:
> I added the warped nether ship in #10648 Not sure why that is not reflected here (it would be an update rather than an addition)

It doesn't reflect because this branch is based on an outdated branch (that's why there are conflicts too).
Please, remove things like the reload anchor and some other special things for the overworld version of the ship then.

Moobien committed:
Kitchen Fixes

Corrected signs on cookery

Raycoms commented:
Closing, in favor of new one with fixes =D
